tools/pfe-tools/dev-server/config.ts

```diff
@@ -57,7 +57,7 @@ function normalizeOptions(options?: PfeDevServerConfigOptions) {
   config.watchFiles ??= '{elements,core}/**/*.{css,html}';
   config.litcssOptions ??= {
     include: /\.css$/,
-    exclude: /(((fonts|demo)|(demo\/.*))\.css$)|(.*(-lightdom.css$))/,
+    exclude: /(((fonts|demo)|(demo\/.*))\.css$)|(.*(-lightdom.css$))|(.*(-lightdom-shim.css$))/,
```

Comment:
most likely possible to shorten this with another splat in the `-lightdom.css` check? I did an attempt and didn't seem like it wanted to work but my regex foo isn't as strong as i'd like. YMMV

Reply:
shameless plug: https://github.com/bennypowers/nvim-regexplainer

```markdown
Either capture group 1:
  capture group 2:
    Either capture group 3:
      Either `fonts` or `demo` or capture group 4:
      `demo/`
      **ANY** (_>= 0x_)
  `.css`
  **END**, capture group 5:
  **ANY** (_>= 0x_)
  capture group 6:
    `-lightdom`
    **ANY**
    `css`
    **END**, or capture group 7:
  **ANY** (_>= 0x_)
  capture group 8:
    `-lightdom-shim`
    **ANY**
    `css`
    **END**
```
I think we can simplify this to:

```js
/**
 * Either `fonts/`, `demo/`, or `-lightdom`
 * non-capturing group (_optional_):
 *   `-shim`
 * `.css`
 * **END**
 */
/fonts\/|demo\/|-lightdom(?:-shim)?\.css$/
```